electron/app/locales/en/webui.json

```diff
@@ -662,6 +662,9 @@
   "ingress-design-ingress-route-traefik-mw-label": "Traefik Middleware",
   "ingress-design-ingress-route-traefik-mw-help": "Customize Traefik Middlewares Object",
 
+  "ingress-design-ingress-routes-getting-target-service-title" : "Retrieving existing target service details",
+  "ingress-design-ingress-routes-getting-target-service-error-message": "Error in getting existing service details: {{error}} from namespace {{namespace}}",
```

Comment:
Change the format of this message so that `: {{error}}` is at the end of the message.

For example: "Failed to get existing service details from Kubernetes namespace {{namespace}}: {{error}}."